packages/dd-trace/src/profiling/profiler.js

```diff
@@ -15,19 +13,16 @@ class Profiler extends EventEmitter {
     if (!config.enabled) return
 
     this._logger = config.logger
-
-    if (!semver.satisfies(process.version, '>=10.12')) {
-      this._logger.error('Profiling could not be started because it requires Node >=10.12')
-      return this
-    }
-
     this._enabled = true
 
     try {
       const mapper = config.sourceMap ? new SourceMapper() : null
 
       for (const profiler of config.profilers) {
-        profiler.start({ mapper }) // TODO: move this outside of profilers
```

Comment:
Keep the comment to not forget to eventually do this when reintroducing source map support.